reader/processor/processor.go

```diff
@@ -49,6 +49,39 @@ func ProcessFeedEntries(store *storage.Storage, feed *model.Feed) {
 		// The sanitizer should always run at the end of the process to make sure unsafe HTML is filtered.
 		entry.Content = sanitizer.Sanitize(entry.URL, entry.Content)
 	}
+
+	filterFeedEntries(feed)
```

Comment:
Filtering entries should happen before crawling them (just above). There is no need to crawl web pages for entries filtered out.

The current logic loop through each entries 3 times. The filtering can probably done in the existing loop.

Reply:
@fguillot makes sense, I moved the func above the crawling loop :)